actions/lookup_remote.yaml

```diff
@@ -0,0 +1,120 @@
+---
+name: lookup_remote
+runner_type: remote-shell-script
```

Comment:
I think this will have the same problem as epp_render, where it won't work on a remote host, because it won't copy across the dependencies?